app/assets/scripts/views/location/averages-card.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { PropTypes as T } from 'prop-types';
+import styled from 'styled-components';
+import LoadingMessage from '../../components/loading-message';
+import Card from '../../components/card';
+import Table from '../../components/table';
+import { shortenLargeNumber } from '../../utils/format';
+
+const initData = {
+  pollutant: {
+    values: [],
+    formatHeader: v => v.toUpperCase(),
+    style: {
+      color: 'black',
+      fontWeight: 700,
+    },
+  },
+  avg: {
+    values: [],
+    formatHeader: v => v.toUpperCase(),
+    formatCell: shortenLargeNumber,
+    style: {
+      textAlign: 'center',
+    },
+  },
+  count: {
+    values: [],
+    formatHeader: v => v.toUpperCase(),
+    formatCell: shortenLargeNumber,
+    style: {
+      textAlign: 'center',
+    },
+  },
+};
+
+const StyledLoading = styled(LoadingMessage)`
+  grid-column: 1 / 7;
+`;
+const ErrorMessage = styled.div`
+  grid-column: 1 / 7;
+`;
+
+// TODO This function currently just extracts the first data available for each pollutant
```

Comment:
What is the task to complete here?

Reply:
We need to confirm what data this table should display. The endpoint returns either all available average data, or a selected date range. For now this just uses the most recent result (also the version of the api that is currently connected is only available for a single pollutant. cc @rwegener2

Reply:
I see, can we add a bit of this context to the comment so it's easier for us to pick up later?